locales/en-US/webextension.properties

```diff
@@ -24,6 +24,9 @@ notificationLinkCopiedTitle = Link Copied
 # modifier key used in the shortcut (do not translate it): for example, Ctrl-V
 # on Windows systems.
 notificationLinkCopiedDetails = The link to your shot has been copied to the clipboard. Press {meta_key}-V to paste.
+copyScreenshot = Copy
```

Comment:
This string needs a localization comment, since it's unclear if it's a noun or a verb without context.